lms/djangoapps/instructor_task/tasks_helper/module_state.py

```diff
@@ -221,6 +225,75 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
 
 
 @outer_atomic
+def override_problem_score_module_state(xmodule_instance_args, module_descriptor, student_module, task_input):
+    '''
+    '''
+    # unpack the StudentModule:
+    course_id = student_module.course_id
+    student = student_module.student
+    usage_key = student_module.module_state_key
+
+    with modulestore().bulk_operations(course_id):
+        course = get_course_by_id(course_id)
+        # TODO: Here is a call site where we could pass in a loaded course.  I
```

Comment:
Is there a ticket that would address this?  If not, you can create one and reference it in this TODO (I think that's the pattern we use for TODOs, right?).